index.js

```diff
@@ -80,37 +60,30 @@ function fromSource(source) {
  * @param  {Object} options
  * @return {String}
  */
-function sync(file, options) {
-  if (!file) {
-    throw new Error('filename missing');
-  }
-  var fileSystem = options ? (options.fileSystem || fs) : fs;
+function sync(file, options = {}) {
+  if (!file) throw new Error('filename missing');
+
+  const fileSystem = options.fileSystem ? options.fileSystem : fs;
   const data = fileSystem.readFileSync(file, 'utf8');
-  return fromSource(data.toString());
```

Comment:
Remove `toString()`; since we specify the encoding in `readFileSync` this should always be a string AFAICT